### DISCUSSION TITLE
The config file (data/config/extensions.conf.php) isn't writable by the web server :(

### DISCUSSION THREAD

**Discussion by s-erasmus** [shish/shimmie2#1150] [Q&A] [answered]
I'm not sure if this is the appropriate place to ask, please let me know if I should delete this.

When I try to enable additional extentions via the "Extension Manager" I get the message "The config file (data/config/extensions.conf.php) isn't writable by the web server :(" 

I am not sure how to rectify this, the only file in my config folder is "shimmie.conf.php" as well, I tried to just create an empty file with that name but nothing changes.

**Comment by shish** [comment, marked as answer]
The answer here depends almost entirely on your system setup -- OS, web server, what user the web server runs as, what user owns the install folder, whether this is running on docker or VM or bare metal

Though in any case - would I be right in guessing that it managed to install (and create `data/config/shimmie.conf.php`) with no problems? It would be very weird to be able to create one file, but not another file in the same folder, unless something else changed in between installation time and now...

**Reply by s-erasmus** [reply]
you would be correct, I had moved the entire folder to another on the same drive after I had created and ran the server at least once. I redid the installation process and it's working now which I should have tried in the first place, apologies for wasting your time.

In any case it's installed on windows 10 with sqlite3 on bare metal, made and run with the same windows acocunt if thats what you meant if for some reason you still wanted to know.
